database/factories/ResourceFactory.php

```diff
@@ -13,5 +13,6 @@
         'type' => $faker->randomElement(['video', 'course', 'audio', 'book', 'article']),
         'module_id' => factory(Module::class),
         'language' => $faker->randomElement(['en', 'es']),
+        'os' => $faker->randomElement(['any', 'osx', 'linux', 'windows']),
```

Comment:
I think we'll probably want to make an enum for this. 

I haven't yet 100% decided how I want to build the enums but here's what I've been thinking:

```php
<?php

namespace App;

use Illuminate\Contracts\Support\Arrayable;

abstract class Enum implements Arrayable
{
    abstract public function keys();

    public function values()
    {
        return collect($this->keys())->map(function ($key) {
            return static::$key;
        })->toArray();
    }

    public function toArray()
    {
        return collect($this->keys())->mapWithKeys(function ($key) {
            return [$key => static::$key];
        })->toArray();
    }
}
```
And:

```php
<?php
namespace App;

class OperatingSystem extends Enum
{
    const MACOS = 'macos';
    const LINUX = 'linux';
    const WINDOWS = 'windows';

    public static function keys()
    {
        return [
            'MACOS',
            'LINUX',
            'WINDOWS',
        ];
    }
}